_partials/wsl2_vscode.md

```diff
@@ -4,25 +4,66 @@ We use Visual Code Studio for writing code on Windows, because it integrates nic
 
 For this integration to work, you need to uninstall other code editor, like for example SublimeText, as their WSL integration interferes with the integration of VS Code.
 
-Go the [Visual Studio Code page](https://code.visualstudio.com/download) and click on Windows. The tool should start to download automatically. Open the file you have just downloaded. Install it leaving all the default options as they are.
+### Installation
+>\- Go the [Visual Studio Code page](https://code.visualstudio.com/download)  
+>\- Choose the Windows version of VS Code
 
-To help VS Code interact with your terminal, we need to add the extension **Remote Development**. Go to the [extension page](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.vscode-remote-extensionpack) and click on **Install**. This will tell you that VS Code is required to install this extension. Click on **Continue**. You will get an alert asking you to open Visual Studio Code. The editor will open to the extension page.
 
-![remote_development](images/remote_development.jpg)
+The tool should start to download automatically.
 
-Click on **Install**.
-Click on **File → Preferences → Keymaps**. Click on **Sublime Text Keymap and Settings Importer**.
-Click on **Install**. Click on **Reload Required**
+>\- Open the file you have just downloaded.  
+>\- Install it with a few options:  
+>    \- Register VS Code as an editor for supported file types  
+>    \- Let the other options as they are
+
+![VS Code install](images/vscode-install.png)
+
+### Connecting VS Code to Ubuntu
+
+- Launch VS Code a first time, then:
+    - Pin it to the taskbar; you will be using it a lot!
+    - You should have a frame at the bottom-right corner of VS Code saying it detects WSL
+    - Accept the installation of the WSL extension for VS Code
+
+:warning: This extension is mandatory for you to be able to work with VS Code!
+
+### VS Code Shortcuts
+
+In VS Code:
+
+
+>\- Click on `File`  
+>\- Click on `Preferences`  
+>\- Click on `Keymaps`  
+>\- Click on `Sublime Text Keymap and Settings Importer`  
+>\- Click on `Install`  
+>\- Click on `Reload Required`
+
+
+### VS Code Extensions
+
+Let's gain time now and add other nice extensions that will help your bootcamp even better in the future.
```

Comment:
This one is on me (I wrote that) but it doesn't make any sense haha.
Let's change it to `Let's gain time now and add other extensions that will be helpful during your Bootcamp`

Reply:
I was not a fan of the wording, but when something is written by a native speaker, I tend to avoid judging it. :smile: